torch/csrc/profiler/python/combined_traceback.cpp

```diff
@@ -84,7 +92,12 @@ struct PythonTraceback : public CapturedTraceback::Python {
       }
     }
     for (const auto& f : to_symbolize) {
-      auto f_code = (PyCodeObject*)f.code;
+      auto f_code = (PyCodeObject*)PyWeakref_GetObject((PyObject*)f.code);
+      // We only hold a weakref to the object, so it might have evaporated.
+      if ((PyObject*)f_code == Py_None) {
+        continue;
```

Comment:
Instead of skipping the frame, wouldn't it be better to print `unknown` or something here?